search/controllers/advanced/tests.py

```diff
@@ -3,7 +3,7 @@
 from unittest import TestCase, mock
 from datetime import date, datetime
 from dateutil.relativedelta import relativedelta
-from werkzeug import MultiDict
+from werkzeug.datastructures import MultiDict
```

Comment:
I wonder if we can get rid of our direct dependence on werkzeug by not using its data structures?

Reply:
There is a separate `multidict` package but I think there is some convenience in having werkzeug bundle a bunch of the HTTP-friendly data structures together.